Library/Homebrew/utils/shared_audits.rb

```diff
@@ -92,6 +92,48 @@ def gitlab_release(user, repo, tag, formula: nil, cask: nil)
     "#{tag} is a GitLab pre-release."
   end
 
+  def codeberg_repo_data(user, repo)
+    @codeberg_repo_data ||= {}
+    @codeberg_repo_data["#{user}/#{repo}"] ||= begin
+      out, _, status = Utils::Curl.curl_output("https://codeberg.org/api/v1/repos/#{user}%2F#{repo}")
```

Comment:
```suggestion
      out, _, status = Utils::Curl.curl_output("https://codeberg.org/api/v1/repos/#{user}/#{repo}")
```

seems to break with %2F for me